src/main/identity/id-attribute-type-service.js

```diff
@@ -1,20 +1,57 @@
 'use strict';
 import fetch from 'node-fetch';
+import { Logger } from 'common/logger';
+import RefParser from 'json-schema-ref-parser';
 import IdAttributeType from './id-attribute-type';
+import { IdAttributeSchema } from './id-attribute-schema';
 
+const log = new Logger('id-attribute-type-service');
 const airtableBaseUrl = 'https://alpha.selfkey.org/marketplace/i/api/';
-
+export const JSON_SCHEMA_EXPIRTION_TIME = 1000 * 60 * 60 * 5; // 5h
```

Comment:
@sk91 I think we should use the same expiration time for all schemas and store it in a global configuration class. Thoughts? Cheers

Reply:
I think this value,  same as the schemaUrl value below, should be defined separately as schema properties. Here we have one value for all but it is only for mvp purposes>

in general schema expiration should be different for each schema